src/module/entities/helper.js

```diff
@@ -11,6 +11,7 @@ const entityTypesHelper = require(MODULES_BASE_PATH + '/entityTypes/helper')
 // const userRolesHelper = require(MODULES_BASE_PATH + "/userRoles/helper");
 // const userProfileService = require("../../generics/");
 const entitiesQueries = require(DB_QUERY_BASE_PATH + '/entities')
+const entityTypeQueries = require(DB_QUERY_BASE_PATH + '/entityTypes')
 // const formService = require(PROJECT_ROOT_DIRECTORY + '/generics/services/form')
```

Comment:
@MallanagoudaB remove these if not used

Reply:
we wan this one

Reply:
@MallanagoudaB remove the commented imports